.circleci/config.yml

```diff
@@ -25,6 +25,17 @@ jobs:
           name: List Simulators
           command: xcrun simctl list
 
+      - run:
+          name: Install Node 12 (Required for Yarn)
+          command: |
+            set +e
+            touch $BASH_ENV
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+            echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
+            echo 'nvm install v12.13.1' >> $BASH_ENV
+            echo 'nvm alias default v12.13.1' >> $BASH_ENV
+
```

Comment:
Looks like adding an `.nvmrc` file has the effect that the first time we run `yarn`, it'll get auto-installed:

```sh
#!/bin/bash --login -eo pipefail
cd Example ; yarn ci:skip-native-if-possible || bundle check || bundle install
N/A: version "12.13.1 -> N/A" is not yet installed.

You need to run "nvm install 12.13.1" to install it before using it.
N/A: version "12.13.1 -> N/A" is not yet installed.

You need to run "nvm install 12.13.1" to install it before using it.
Downloading and installing node v12.13.1...

...
```

Reply:
Nope, it's needed apparently!